entrypoint.sh

```
@@ -185,16 +185,24 @@ if [ "$claude_rc" -ne 0 ] && [ "$claude_rc" -ne 143 ]; then exit "$claude_rc"; f
     claude -p "${CLAUDIO_EVALUATION_PROMPT}" \
       --model "${CLAUDIO_EVALUATION_MODEL:-claude-haiku-4-5-20251001}" \
-      --no-session-persistence \
-      > "${CLAUDIO_RESULT_FILE}"
+      --no-session-persistence)
   then
     echo "ERROR: Failed to evaluate task result"
     exit 1
   fi
 
+  # Extract the verdict line — models sometimes wrap it in extra text
+  local verdict
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)